src/extensions/default/bramble/main.js

```diff
@@ -120,6 +121,13 @@ define(function (require, exports, module) {
 
         // We're all done loading and can pass startup state info back to the host app.
         RemoteEvents.loaded();
+
+        // Show Tutorial Pane by default if `tutorial.html` exists at project load
```

Comment:
If we want to avoid firing a second event, we'd have to make changes to how the `bramble:loaded` event is handled by Thimble. (I couldn't find any references to it in the code however. Any idea where that might be located?)